semconv/v1.12.0/http.go

```diff
@@ -75,7 +76,7 @@ func EndUserAttributesFromHTTPRequest(request *http.Request) []attribute.KeyValu
 // HTTPClientAttributesFromHTTPRequest generates attributes of the
 // http namespace as specified by the OpenTelemetry specification for
 // a span on the client side.
-func HTTPClientAttributesFromHTTPRequest(request *http.Request) []attribute.KeyValue {
+func HTTPClientAttributesFromHTTPRequest(request *http.Request, opts ...config.Option) []attribute.KeyValue {
```

Comment:
Is it really non-backwards compatible though?
The signature changes, but any call with a single argument will still work.

Reply:
My [usual go-to for BC guidance](https://go.googlesource.com/exp/+/master/apidiff/README.md#functions) says no joy as, while call-site compatibility is fine, assignment compatibility would break.